css/styles.css

```diff
@@ -0,0 +1,266 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+    float: center center;
+    background-color: lightgrey;
+    font-family: Merriweather;
+  }
+
+img.background {
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 100%;
+    float: center;
+  }
+
+header  {
+  color: white;
+  background-color: #000000;
+  height: 100px;
+  border: 1px;
```

Comment:
@Schultisal when declaring a border you should have the border <width> <style> <color>. For example ` border: 1px solid red;`  🔨